cpp_generation/modelSchemas/configs/HcConfig.json

```diff
@@ -324,11 +324,11 @@
       }
   },
   "intermediate": {
-    "age15_hiv_pop": {
+    "age15_hivpop": {
       "num_type": "real_type",
       "dims": ["SS::hc2DS", "SS::NS"]
     },
-    "age15_art_pop": {
+    "age15_artpop": {
```

Comment:
Not a big deal, but these names might be more semantically consistent as `age15_h_hivpop` or `h_hivpop_age15_entrants`. (Is that where this is used -- for entrants into the 15+ age population?